sdk/communication/azure-communication-phonenumbers/src/samples/java/com/azure/communication/phonenumbers/FindAndPurchasePhoneNumber.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.phonenumbers;
+
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilities;
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilityType;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchOptions;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchResult;
+import com.azure.communication.phonenumbers.models.PhoneNumberType;
+import com.azure.communication.phonenumbers.models.PhoneNumberAssignmentType;
+import com.azure.communication.phonenumbers.models.PhoneNumberOperation;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.PollResponse;
+
+public class FindAndPurchasePhoneNumber {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
```

Comment:
With samples, we usually put a fake value. E.g.
```
String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
String accessKey = "SECRET";
```

Reply:
to be in sync with other sdks we what to make this a runable example

Reply:
See comment above

Reply:
See comment above

> > In general, since these new sample files are being added in addition to the `ReadmeSamples.java` files, do we plan on having two samples files? Is this completely necessary?
> > I wonder if we can only keep one of the files and point the README snippets to the remaining file. @paolamvhz
> 
> Checking other Sdk and to be in sync we saw that other sdks have some main runnable files to show the principal functions an other sample file which contain snippets for the README file

I

> > In general, since these new sample files are being added in addition to the `ReadmeSamples.java` files, do we plan on having two samples files? Is this completely necessary?
> > I wonder if we can only keep one of the files and point the README snippets to the remaining file. @paolamvhz
> 
> Checking other Sdk and to be in sync we saw that other sdks have some main runnable files to show the principal functions an other sample file which contain snippets for the README file

I see! thanks for the clarification. In this case, can we rename these files to the convention that the `storage` sdk is using? 

The ones that are runnable are named `*Example.java` and the ones that are just code snippets are named `*Snippets.java`